cmd/cagent/main.go

```diff
@@ -105,7 +111,15 @@ func main() {
 
 	handleFlagPrintConfig(*printConfigPtr, cfg)
 
-	setDefaultLogFormatter()
+	if ((serviceInstallPtr == nil) || ((serviceInstallPtr != nil) && (!*serviceInstallPtr))) &&
```

Comment:
looks like https://github.com/jessevdk/go-flags suggests itself as this now look too ugly.
@soupdiver @requilence ?

Reply:
agree. Now the CLI API becomes to look ugly 
<img width="657" alt="image" src="https://user-images.githubusercontent.com/1025231/51424386-9303e200-1bd5-11e9-93e1-ad8409306703.png">

Reply:
I agree, it get's messy.
An opinion on grouping? E.g. `cagent service start` and maybe the other options could also be grouped... do we need backwards compatibility in this case?